SDE (#1884)
arbor-sim/arbor: doc/cpp/simulation.rst

        // Instantiate the simulation.
        arb::simulation sim(recipe, decomp, context);

> All the simulation's constructor arguments apart from the recipe are optional, and will assume

Reviewer: ```suggestion
All the simulation's constructor arguments are optional, except the recipe, and assume
```
Also: are these changes by you? Reads like they may come from an earlier PR by Thorsten, but maybe I'm seeing things.

Author: the arguments have been optional for a while, but it was not mentioned anywhere.
